### PR TITLE
add global stylesheet example

### LINKED ISSUE #544: Importing CSS files?
Sometimes it's nice to break out your CSS into a separate `.css` file. I've tried to do the following:

```
pages/
└── index
    ├── index.css
    ├── index.js
    └── component.js
```

Then in the index.js, I've tried to do:

```js
import css from './index.css'
```

And in next.config.js:

```js
module.exports = {
  webpack: function (config) {
    config.module.loaders = (config.module.loaders || []).concat({
      test: /\.css$/, loader: 'raw'
    })
    return config
  }
}
```

But unfortunately, it keeps giving me:

```
 ERROR  Failed to compile with 1 errors

This dependency was not found in node_modules:

* ./index.css
```

Seems like it's not resolving to the right place for some reason, the local `component.js` works though via `import component from './component.js'`, so I'm not sure what's going on here.

### PR DESCRIPTION
**Pull request by davibe**
I know "global" styles are not a priority for 2.0 release but i think this example really solves it for many users in the mean time.

Fixes #544

### CODE REVIEW

**Comment by nikvm** [comment]
Hey @davibe , thanks for the example.

Just today I was looking to add an external CSS file. I got it working with [`babel-plugin-inline-import`](https://www.npmjs.com/package/babel-plugin-inline-import).

After a quick look into your example I noticed a couple things:
1.  One should inject a string of CSS into a jsx `<style>`-tag with [`dangerouslySetInnerHTML`](https://facebook.github.io/react/docs/dom-elements.html#dangerouslysetinnerhtml) otherwise the string will get escaped (noticeable when using _"_ or _'_ as in `font-family: "Comic Sans", serif`.
2. My understanding is that on the server side code doesn't get bundled with Webpack so your SCSS example might not be working when rendering server side?

**Comment by davibe** [comment]
1. Right, i fixed this.
2. It works fine, i just checked. You can try with `curl` and see the initial page content. Last time i read the next.js sources I understood that webpack is used to generate the files for the client but those are the same files that are loaded and reloaded by the server when it has to render. So webpack+babel transpilation has already occurred for both sides.

**Comment by rauchg** [comment]
You're all doing such awesome work  💖

File: examples/with-global-stylesheet/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.next
```
Comment (timneutkens) [inline examples/with-global-stylesheet/.gitignore]: No other examples include a `.gitignore`

File: examples/with-global-stylesheet/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+README
```
Comment (timneutkens) [inline examples/with-global-stylesheet/README.md]: Can you port this documentation to the same style as https://github.com/zeit/next.js/tree/master/examples/basic-css. All examples have kind of the same readme ❤️

**Review by timneutkens — COMMENTED** [review]
Besides the 2 points I had this looks great, it's a port of your github repo right 👍

**Comment by davibe** [comment]
Yes. I would like to keep it in sync with the repo because i can track issues there and sync from time to time

File: examples/with-global-stylesheet/.gitignore (hunk 1 above)
Comment (davibe) [inline examples/with-global-stylesheet/.gitignore]: it does not hurt but i'll remove it

File: examples/with-global-stylesheet/README.md (hunk 2 above)
Comment (davibe) [inline examples/with-global-stylesheet/README.md]: done

**Review by timneutkens — APPROVED** [review]
Looks good 👍

**Pull request merged**